pr-template.md

```diff
@@ -1,35 +1,30 @@
 #### What does this PR do? (please provide any background)
 
-
 #### What tests does this PR have?
 
-
 #### How can this be tested?
 
+#### What domains are affected?
 
-#### Screenshots / Screencast
+#### Any tech debt?
 
+#### Screenshots / Screencast
 
 #### What gif best describes how you feel about this work?
-![](url)
-
----
+![]()
 
 - [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.
 
-#### Reviewers
-
-**Review 1**
-- [ ] :+1:
-
-**Review 2**
-- [ ] :+1:
+---
 
-**Review 3**
+**Reviewer**
 - [ ] :+1:
+- [ ] This PR need any additional reviewers!
 
 By adding a +1 you are confirming you have...
```

Comment:
Does this and the check on line 16 add any real value. I think it just adds noise to the description and people see it too often to actually shock them into changing their behaviour. 

It feels a little like the immigrations question "Do you or have you had any links to Nazis".